manticore/ethereum.py

```diff
@@ -238,6 +238,19 @@ def _get_src(self, address, pc):
         return self.manticore.get_metadata(address).get_source_for(pc)
 
 
+class VerboseTrace(Plugin):
+    def will_evm_execute_instruction_callback(self, state, instruction, arguments):
+        world = state.platform
+        current_vm = world.current_vm
+        str_trace = state.context.get('str_trace', [])
+        str_trace.append(str(current_vm))
+        state.context['str_trace'] = str_trace
```

Comment:
Those 3 lines can be changed to: `state.setdefault('str_trace', []).append(str(current_vm))`